server/game/cards/02_SHD/events/LookTheOtherWay.ts

```diff
@@ -25,7 +25,7 @@ export default class LookTheOtherWay extends EventCard {
                         [`Exhaust ${context.targets.targetUnit.title}`]: AbilityHelper.immediateEffects.exhaust({
                             target: context.targets.targetUnit,
                         }),
-                        ['Pay 2 resources']: AbilityHelper.immediateEffects.payResourceCost({
+                        ['Pay 2 resources']: AbilityHelper.immediateEffects.exhaustResources({
```

Comment:
In this case I think it should be `payResourceCost`, unless that caused some issue